kfs-core/src/main/resources/edu/arizona/kfs/workflow/CashReceiptDocument.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data xmlns="ns:workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+  <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+    <documentType>
+      <name>CR</name>
+      <parent>FP</parent>
+      <label>Cash Receipt</label>
+      <docHandler>${kfs.url}/financialCashReceipt.do?methodToCall=docHandler</docHandler>
+      <helpDefinitionURL>default.htm?turl=WordDocuments%2Fcashreceipt.htm</helpDefinitionURL>
+      <active>true</active>
+      <routingVersion>2</routingVersion>
+      <routePaths>
+        <routePath>
+          <start name="AdHoc" nextNode="CashManagement" />
```

Comment:
Misty has confirmed that the ObjectSubTypeCode routing should be before the CashManagement routing.